packages/react/src/components/Combobox/index.ts

```diff
@@ -1,4 +1,22 @@
-export { EXPERIMENTAL_Combobox } from './Combobox';
-export type { ComboboxFilter } from './types/ComboboxFilter';
-export type { ComboboxOption } from './types/ComboboxOption';
-export type { ComboboxProps } from './Combobox';
+import { Combobox as ComboboxRoot } from './Combobox';
+import { ComboboxOption } from './Option/Option';
+import { ComboboxEmpty } from './Empty/Empty';
+
+type ComboboxComponent = typeof ComboboxRoot & {
+  Option: typeof ComboboxOption;
+  Empty: typeof ComboboxEmpty;
+};
+
+const Combobox = ComboboxRoot as ComboboxComponent;
+
+Combobox.Option = ComboboxOption;
+Combobox.Empty = ComboboxEmpty;
+
+Combobox.Option.displayName = 'Combobox.Option';
+Combobox.Empty.displayName = 'Combobox.Empty';
```

Comment:
Forgot to add `EXPERIMENTAL_` to displayName. This is what is shown in react devtools and storybooks "show code"